<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdapterElmWidget</class>
 <widget class="QWidget" name="AdapterElmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ELM327</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/images/adapter/icon_elm.png</normaloff>:/images/adapter/icon_elm.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="source.qrc">:/images/adapter/photo_elm.jpg</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;html&gt;
&lt;head/&gt;
&lt;body&gt;

&lt;h2 style=&quot;padding:0; margin:0;&quot;&gt;ELM327 Adapter&lt;/h2&gt;

&lt;hr&gt;

&lt;p style=&quot;padding:0; margin-top:0;&quot;&gt;The ELM327 is a programmed microcontroller produced by ELM Electronics for translating the on-board diagnostics (OBD) interface found in most modern cars.&lt;/p&gt;

&lt;p style=&quot;padding:0; margin-bottom:0;&quot;&gt;The ELM327 command protocol is one of the most popular PC-to-OBD interface standards and is also implemented by other vendors.&lt;/p&gt;

&lt;p style=&quot;margin-top:0; text-align:right;&quot;&gt;&amp;mdash; &lt;a href=&quot;https://en.wikipedia.org/wiki/ELM327&quot;&gt;Wikipedia&lt;/a&gt;&lt;/p&gt;

&lt;/body&gt;
&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="_radioWiFi">
        <property name="text">
         <string>Connect over Wi-Fi</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Address:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="_editWiFiAddress">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="inputMask">
           <string>000.000.000.000</string>
          </property>
          <property name="text">
           <string>192.168.0.10</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="_editWiFiPort">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>35000</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="_buttonWiFiDefault">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_buttonWiFiCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Check</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_radioUart">
        <property name="text">
         <string>Connect over USB/RS232/Bluetooth</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="_comboUartPort">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Baud rate:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="_comboUartBaud">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="_buttonUartRefresh">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_buttonUartCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Check</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>_radioWiFi</sender>
   <signal>toggled(bool)</signal>
   <receiver>_editWiFiAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioWiFi</sender>
   <signal>toggled(bool)</signal>
   <receiver>_editWiFiPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioWiFi</sender>
   <signal>toggled(bool)</signal>
   <receiver>_buttonWiFiDefault</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioWiFi</sender>
   <signal>toggled(bool)</signal>
   <receiver>_buttonWiFiCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioUart</sender>
   <signal>toggled(bool)</signal>
   <receiver>_comboUartPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioUart</sender>
   <signal>toggled(bool)</signal>
   <receiver>_comboUartBaud</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioUart</sender>
   <signal>toggled(bool)</signal>
   <receiver>_buttonUartRefresh</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_radioUart</sender>
   <signal>toggled(bool)</signal>
   <receiver>_buttonUartCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
